courses-1.1

tcp opened the pull request:
Updates courses to the 1.1 version

Adds Simulcast support
Fixes bug where mentors were unable to review a project in the dashboard
Several UI and UX improvements
Improves page navigation
Makes more course statistics available

Issues
- Simulcast support #132 
- Impossible to finish course (staging) #124
- Next button is not working (staging) #123
- Mobile view - courses content should not be cropped by header and footer #121 
- Mentors should review one project per turn #115 
- Unable to access previous reviews and submissions for projects which were "failed" #101 
- Unable to review project #99 
- In small device navbar is hiding the content #98 
- Course complete page not working #96 
- Get all users and reorder and push #87 
- Refactor helpers tests #63
- Rewrite the helpers functions #62 
- Keep a running tally of the number of reviewed submissions and resigned submissions #61
- Keep a running tally of the number of project submissions in the queue #60 
- Add tests for remaining courses helpers functions #56
- Redo the page changing logic #53 
- Store the "number" of a user according to when they signed up #51

gavinuhma commented:
looks good to me @moodlep 

the inline version can be a separate pr: https://github.com/OpenMined/PySyft/pull/673#pullrequestreview-80729167

ITsolution-git committed:
Add new permission gate

ITsolution-git committed:
new getNextAvailablePage

ITsolution-git committed:
Get all users and reorder and push

ITsolution-git committed:
Add calculation clouding functions

ITsolution-git committed:
Change the batch size; Timeout and memory on addUNumToAllUsers

ITsolution-git committed:
Change the function name properly

ITsolution-git committed:
Add the numbers in view

ITsolution-git committed:
Clean up

ITsolution-git committed:
finalize permission gate; add some tests

tcp committed:
Merge pull request #85 from OpenMined/53-permission-gate

Permissions gate refactor

ITsolution-git committed:
mobile scale 1 and mobile optimization

tcp committed:
Merge pull request #87 from OpenMined/51-unique-number-user

Get all users and reorder and push

ITsolution-git committed:
Resolve conflicts

tcp committed:
Merge pull request #121 from OpenMined/98-mobile-content-header-footer-size

fix: course content was shown under the header and the footer

tcp committed:
Merge pull request #89 from OpenMined/60-project-submissions

60 project submissions

tcp committed:
sanity env

tcp committed:
Revert "sanity env"

This reverts commit 5e94a4d3aeabfcd35754ac53c0cc83a1b322a629.

tcp committed:
add secret

ITsolution-git committed:
Fix footer height problem

ITsolution-git committed:
Fix redirection issue

tcp committed:
Merge pull request #128 from OpenMined/123-fix-footer-height

Fix footer height problem

tcp committed:
Merge pull request #131 from OpenMined/fix/homepage-courses-list

Fix/homepage courses list

tcp committed:
fix: quick fixes that allows loading simulcast courses in students dashboard

tcp committed:
remove debugger

tcp committed:
fix: remove whitespace.........

tcp committed:
Merge pull request #129 from OpenMined/124-fix-redirection-issue

Fix redirection issue

tcp committed:
fix: course overview simulcast partial support

tcp committed:
feat: course card showing simulcast

tcp committed:
feat: dashboard course progress card with simulcast

tcp committed:
feat: course overview

tcp committed:
feat: helpers guard

tcp committed:
feat: simulcast last lesson complete start

tcp committed:
feat: simulcast last lesson complete working

tcp committed:
feat: sanity updates

tcp committed:
Merge pull request #133 from OpenMined/feature/simulcast-support

feat: adds Simulcast support to courses 1.0

tcp committed:
fix: remove console log

tcp committed:
fix: build

tcp committed:
refactor: using typeof for project description calls and only displaying projects that have titles (simulcast)

tcp committed:
fix: enhanced NumberedAccording simulcast support

tcp committed:
fix: do not show projects/lessons without a title

This is super paleative. We need to enforce this in a much stricter
sense, perhaps by filtering these lessons or projects out from the
sanity request.

tcp committed:
fix: dashboard progress card title will link to course syllabus

tcp committed:
Merge pull request #135 from OpenMined/fix/simulcast-review

fix: Simulcast design changes
